apps/bfd-server/bfd-server-war/src/main/java/gov/cms/bfd/server/war/stu3/providers/SNFClaimTransformer.java

```diff
@@ -345,7 +345,10 @@ private static ExplanationOfBenefit transformClaim(SNFClaim claimGroup) {
           claimLine.getTotalChargeAmount(),
           claimLine.getNonCoveredChargeAmount(),
           BigDecimal.valueOf(claimLine.getUnitCount()),
-          claimLine.getNationalDrugCodeQuantity(),
+          claimLine.getNationalDrugCodeQuantity().isPresent()
+              ? Optional.of(
+                  BigDecimal.valueOf(claimLine.getNationalDrugCodeQuantity().get().longValue()))
+              : Optional.empty(),
```

Comment:
IMO, This is near the line of complexity where i'd recommend breaking it out of a ternary and into a statement, then pass the result of the statement to this line.

Reply:
agreed; will break apart retrieval of the long value prior to the ternary.